ldms/src/sampler/lustre_mdc/NOTES

```diff
@@ -0,0 +1,66 @@
+Read data found in
```

Comment:
I do. It's aimed at maintainers rather than users, so not included in man page. Hoping to speed up things for whoever updates when lustre changes its metrics again.